src/SparkTestBase.sol

```diff
@@ -57,12 +74,29 @@ abstract contract SparkTestBase is ProtocolV3TestBase {
     IPoolAddressesProvider         internal poolAddressesProvider;
     IPoolAddressesProviderRegistry internal poolAddressesProviderRegistry;
     IPoolConfigurator              internal poolConfigurator;
+    IAuthority                     internal authority;
+    ISparkLendFreezerMom           internal freezerMom;
+
+    address constant AUTHORITY             = 0x0a3f6849f78076aefaDf113F5BED87720274dDC0;
+    address constant FREEZER_MOM           = 0xFA36c12Bc307b40c701D65d8FE8F88cCEdE2277a;
+    address constant INCENTIVES_CONTROLLER = 0x4370D3b6C9588E02ce9D22e684387859c7Ff5b34;
+
+    address constant DAI  = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    address constant WETH = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    address constant MKR  = 0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2;
+
+    address constant SPELL_FREEZE_ALL = 0xA67d62f75F8D11395eE120CA8390Ab3bF01f0b8A;
+    address constant SPELL_FREEZE_DAI = 0x0F9149c4d6018A5999AdA5b592E372845cfeC725;
+    address constant SPELL_PAUSE_ALL  = 0x216738c7B1E83cC1A1FFcD3433226B0a3B174484;
+    address constant SPELL_PAUSE_DAI  = 0x1B94E2F3818E1D657bE2A62D37560514b52DB17F;
```

Comment:
This should all be added to `SparkEthereumTestBase` instead of the general one. We want this to work cross-chain.

Reply:
Agreed